frontend/src/components/funding/FundingAllocationInfoTable.vue

```diff
@@ -0,0 +1,91 @@
+        <v-col cols="2" class="table-cell">Does Not Need Ministry Approval</v-col>
+        <v-col cols="2" class="table-cell">Does Not Need Ministry Approval</v-col>
+        <v-col cols="2" class="table-cell">Need Ministry Approval</v-col>
+        <v-col cols="2" class="table-cell">Does Not Need Ministry Approval</v-col>
+        <v-col cols="2" class="table-cell">Need Ministry Approval</v-col>
+      </v-row>
+      <v-row no-gutters>
+        <v-col cols="2" class="table-header"><strong>Sending Funds</strong></v-col>
+        <v-col cols="2" class="table-cell">Need Ministry Approval</v-col>
+        <v-col cols="2" class="table-cell">Does Not Need Ministry Approval</v-col>
+        <v-col cols="2" class="table-cell">Does Not Need Ministry Approval</v-col>
+        <v-col cols="2" class="table-cell">Does Not Need Ministry Approval</v-col>
+        <v-col cols="2" class="table-cell">Need Ministry Approval</v-col>
+      </v-row>
+    </div>
+    <div v-else class="table">
+      <v-row no-gutters>
+        <v-col cols="4" class="table-header"><strong>Funding Envelopes</strong></v-col>
+        <v-col cols="4" class="table-header"><strong>Receive Funds</strong></v-col>
+        <v-col cols="4" class="table-header"><strong>Sending Funds</strong></v-col>
+      </v-row>
+      <v-row no-gutters>
+        <v-col cols="4" class="funding-envelope-types"><strong>Facility</strong></v-col>
+        <v-col cols="4" class="table-cell">Does Not Need Ministry Approval</v-col>
+        <v-col cols="4" class="table-cell">Need Ministry Approval</v-col>
+      </v-row>
+      <v-row no-gutters>
+        <v-col cols="4" class="funding-envelope-types"><strong>Operational</strong></v-col>
+        <v-col cols="4" class="table-cell">Does Not Need Ministry Approval</v-col>
+        <v-col cols="4" class="table-cell">Does Not Need Ministry Approval</v-col>
+      </v-row>
+      <v-row no-gutters>
+        <v-col cols="4" class="funding-envelope-types"><strong>Administration</strong></v-col>
+        <v-col cols="4" class="table-cell">Need Ministry Approval</v-col>
+        <v-col cols="4" class="table-cell">Does Not Need Ministry Approval</v-col>
+      </v-row>
+      <v-row no-gutters>
+        <v-col cols="4" class="funding-envelope-types"><strong>Programming</strong></v-col>
+        <v-col cols="4" class="table-cell">Does Not Need Ministry Approval</v-col>
+        <v-col cols="4" class="table-cell">Does Not Need Ministry Approval</v-col>
+      </v-row>
+      <v-row no-gutters>
+        <v-col cols="4" class="funding-envelope-types"><strong>Human Resources</strong></v-col>
+        <v-col cols="4" class="table-cell">Need Ministry Approval</v-col>
+        <v-col cols="4" class="table-cell">Need Ministry Approval</v-col>
+      </v-row>
+    </div>
+  </v-container>
+</template>
+
+<script>
+export default {
+  name: 'FundingAllocationInfoTable',
+}
+</script>
+
+<style scoped>
+.table {
```

Comment:
I know this is scoped but can we use a more descriptive name so that it doesn't overlap with an html element?

Reply:
it's updated.